--- gq-0.3.0/src/mainwin.c	Fri Nov 10 17:43:28 2000
+++ gq-0.3.0-delete//src/mainwin.c	Fri Nov 10 17:42:28 2000
@@ -150,12 +150,11 @@
 	  GList *searchbase_list;
 
 	  server = server_by_name(config.cur_servername);
-	  if (server)
+	  if (server) {
 	       gtk_entry_set_text(GTK_ENTRY(GTK_COMBO(searchbase_combo)->entry), server->basedn);
-
-	  searchbase_list = NULL;
-	  searchbase_list = g_list_append(searchbase_list, server->basedn);
-	  gtk_combo_set_popdown_strings(GTK_COMBO(searchbase_combo), searchbase_list);
+	       searchbase_list = g_list_append(NULL, server->basedn);
+	       gtk_combo_set_popdown_strings(GTK_COMBO(searchbase_combo), searchbase_list);
+	  }
      }
 }
 
