--- gq-0.3.0/src/schemabrowse.c	Fri Nov 10 17:39:19 2000
+++ gq-0.3.0/src/schemabrowse.c	Fri Nov 10 17:39:49 2000
@@ -1612,7 +1612,7 @@
 			  GdkEventButton *event, gpointer data)
 {
      GList *list;
-     LDAPObjectClass *oc;
+     LDAPObjectClass *oc = NULL;
      struct ldapserver *server;
      char *ocname;
 
@@ -1648,7 +1648,7 @@
 			  GdkEventButton *event, gpointer data)
 {
      GList *list;
-     LDAPAttributeType *at;
+     LDAPAttributeType *at = NULL;
      struct ldapserver *server;
      char *attrname;
 
@@ -1684,7 +1684,7 @@
 			  GdkEventButton *event, gpointer data)
 {
      GList *list;
-     LDAPMatchingRule *mr;
+     LDAPMatchingRule *mr = NULL;
      struct ldapserver *server;
      char *mrname;
 
