diff -Nurb gq-1.0beta1/src/util.c n.gq-1.0beta1/src/util.c
--- gq-1.0beta1/src/util.c	2003-11-03 23:28:24.000000000 +0100
+++ n.gq-1.0beta1/src/util.c	2004-11-27 17:40:04.238821832 +0100
@@ -1906,7 +1906,8 @@
    Masarati <masarati@aero.polimi.it>. 
 */
 
-char **gq_ldap_explode_dn(const char *dn, int dummy)
+char **(*gq_ldap_explode_dn)(const char *dn, int dummy) = ldap_explode_dn;
+/* char **gq_ldap_explode_dn(const char *dn, int dummy)
 {
      int i, rc;
      LDAPDN *parts;
@@ -1915,9 +1916,9 @@
      rc = ldap_str2dn(dn, &parts, LDAP_DN_FORMAT_LDAPV3);
 
      if (rc != LDAP_SUCCESS || parts == NULL) return NULL;
-
+*/
      /* count DN elements */
-     for( i = 0 ; parts[i] ; i++ ) ;
+/*     for( i = 0 ; parts[i] ; i++ ) ;
      v = (char **) calloc((i + 2), sizeof(char*));
 
      for( i = 0 ; parts[i] ; i++ ) {
@@ -1926,7 +1927,7 @@
      } 
      return v;
 }
-     
+*/   
 void gq_exploded_free(char **exploded_dn)
 {
      if (exploded_dn) free(exploded_dn);
diff -Nurb gq-1.0beta1/src/util.h n.gq-1.0beta1/src/util.h
--- gq-1.0beta1/src/util.h	2003-11-03 23:28:24.000000000 +0100
+++ n.gq-1.0beta1/src/util.h	2004-11-27 17:42:31.008509448 +0100
@@ -122,7 +122,8 @@
 #define gq_ldap_explode_dn ldap_explode_dn
 #define gq_exploded_free ldap_value_free
 #else
-char **gq_ldap_explode_dn(const char *dn, int dummy);
+//char **gq_ldap_explode_dn(const char *dn, int dummy);
+char **(*gq_ldap_explode_dn)(const char *dn, int dummy);
 void gq_exploded_free(char **exploded_dn);
 #endif
 
